client/components/EventDetailsComponent.js

```diff
@@ -242,7 +241,7 @@ class EventDetailsComponent extends React.Component {
     const displayTimes = {};
     const formatStr = this.state.days ? 'dddd' : 'DD MMM';
 
-    event.participants.forEach(user => {
+    event.participants.forEach((user) => {
```

Comment:
Airbnb's style guide enforces paran's on the arguments of any arrow functions that has a block enclosed by braces.